### PR TITLE
updated v1.5.0

### PR DESCRIPTION
**Pull request by dhirschfeld**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,13 +18,13 @@ build:
...
-    - python
+    - python >=3.6
   run:
     - azure-core <2.0.0,>=1.0.0
     - cryptography >=2.1.4
     - msal <2.0.0,>=1.3.0
     - msal_extensions >=0.2.2,<0.3.0
-    - python
+    - python >=3.6
     - six >=1.6
```
Comment (dhirschfeld) [inline recipe/meta.yaml:28]:
```python
    install_requires=[
        "azure-core<2.0.0,>=1.0.0",
        "cryptography>=2.1.4",
        "msal<2.0.0,>=1.6.0",
        "msal-extensions~=0.3.0",
        "six>=1.6",
    ],
```
https://github.com/Azure/azure-sdk-for-python/blob/0d70f5758f4d6083fc1deb57012c94e15d0f79e8/sdk/identity/azure-identity/setup.py#L74-L80

**Comment by dhirschfeld** [comment]
That seems to have done the job! 🎉

**Comment by dhirschfeld** [comment]
Ping @marcelotrevisani! Would be great if you've got time to merge this. I'm happy to lend a hand going forward...

**Review by marcelotrevisani — APPROVED** [review]

**Comment by marcelotrevisani** [comment]
> Ping @marcelotrevisani! Would be great if you've got time to merge this. I'm happy to lend a hand going forward...

Sorry for my delay, I got sick in the past days and also because of the holidays it got delayed

**Comment by dhirschfeld** [comment]
> Sorry for my delay

Not a problem - being merged in under a day is very fast! So, thanks for the merge. I took the opportunity to update a few other repos I'm using too so I can help out in future...

**Pull request merged**